src/client/components/pages/entity-revisions.js

```diff
@@ -24,14 +26,28 @@ const Col = bootstrap.Col;
 const ListGroup = bootstrap.ListGroup;
 const ListGroupItem = bootstrap.ListGroupItem;
 
+/**
+* The class is derived from React Component base class and
+* renders the 'Entity Revisions' page.
+*/
 class EntityRevisions extends React.Component {
 
+/**
+* Binds the class methods to their respective data.
+* @constructor
+* @param {object} props - Object of properties passed to the component
+*/
 	constructor(props) {
 		super(props);
 		this.renderHeader = this.renderHeader.bind(this);
 		this.renderRevision = this.renderRevision.bind(this);
 	}
 
+/**
+* Renders the Header of the page consisting of Entity name and
+* disambiguation comment.
+* @returns {ReactElement} a HTML document which is a a part of Revision page
```

Comment:
Repeated "a" in this comment. Also it would make more sense if you wrote "is a part of the Revision page"